### PR TITLE
Adding val-1e case

### PR DESCRIPTION
**Pull request by singhgp4321**
#12

### CODE REVIEW

**Comment by singhgp4321** [comment]
@lindsayad @cticenhour For some reason the concentration profile does not evolve with time. The solution is incorrect. Do you have any insight?

**Comment by lindsayad** [comment]
@singhgp4321 I'm going to be on vacation for the next two and a half weeks. There's a good chance I won't be able to look at this until I'm back. Sorry for the delay!

**Comment by lindsayad** [comment]
@singhgp4321 sorry it took me so long to get around to this. Please see the commit I just pushed. I think things look pretty good now

**Comment by singhgp4321** [comment]
@cticenhour

**Comment by singhgp4321** [comment]
@cticenhour @lindsayad Tests with threads and parallel are failing due to csvdiff. I've tried tightening the nl_rel_tol but that does not work.

**Comment by cticenhour** [comment]
I was able to duplicate the issue on moosebuild, and think I have some things to try tomorrow. Might reach out if I get stuck.

**Comment by lindsayad** [comment]
@singhgp4321 why did you choose the point that you did for the postprocessor?

**Comment by lindsayad** [comment]
It's possible that the `PointValue` postprocessor is not robust when the length scale is as small as it is in this test. For instance I had to make sure to match IDs instead of performing geometric topology based matching with `Exodiff` because it complained that the element centroids were so close together there were multiple matches. Anyway with my added commit, serial, parallel, and threads are passing for me locally. We'll see what CIVET says

File: test/tests/ver-1e/ver-1e.i [hunk 1]
```diff
@@ -75,7 +68,9 @@
   petsc_options_iname = '-pc_type -pc_hypre_type'
   petsc_options_value = 'hypre boomeramg'
   scheme = 'crank-nicolson'
-  nl_rel_tol = 1e-9
+  nl_rel_tol = 1e-50
```
Comment (cticenhour) [inline test/tests/ver-1e/ver-1e.i:71]: This might be the tightest relative tolerance I have ever seen 😮

Comment (lindsayad) [reply test/tests/ver-1e/ver-1e.i:71]: Haha it just ensures that the `nl_abs_tol` will be used

Comment (cticenhour) [reply test/tests/ver-1e/ver-1e.i:71]: Ah right - just like we do for `abs` default. I would suggest that a comment is entered here for the novice MOOSE/TMAP8 user who might look at this test as an example and wonder why we're doing it.

File: doc/content/verification/ver-1e.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# val-1e
+
+# Diffusion in a composite layer
+
+# Depleting Source Problem
```
Comment (cticenhour) [inline doc/content/verification/ver-1e.md:5]:
```suggestion
# ver-1e

# Diffusion in Composite Material Layers
```

@singhgp4321 assuming that the depleting source title snuck in from a copy-paste. Feel free to disregard if both headings are necessary.

**Review by cticenhour — CHANGES_REQUESTED** [review]
Found one more thing that might need to change before we merge, but it looks like the last round of testing was successful. I think we'll be ready to go after this if @singhgp4321 is happy with the robustify-ing!

**Comment by singhgp4321** [comment]
Sorry for this delayed response.

@lindsayad That particular point was chosen based on the documentation for TMAP4 or TMAP7: they used that point for verification and I just to be consistent with them. Thanks for fixing the issue.

File: doc/content/verification/ver-1e.md (hunk 2 above)
Comment (singhgp4321) [reply doc/content/verification/ver-1e.md]: @cticenhour Yes, a copy-paste error; just removed it.

**Review by cticenhour — APPROVED** [review]

**Pull request merged**